docs/reference/docs/bulk.asciidoc

```diff
@@ -106,6 +106,10 @@ Python::
 
     See http://elasticsearch-py.readthedocs.org/en/master/helpers.html[elasticsearch.helpers.*]
 
+JavaScript::
+
+    See https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/client-helpers.html[client.helpers.*]
```

Comment:
We try to avoid linking to the `current` branch as it periodically changes, which can break links.

Instead, you can use the `{jsclient}` attribute, which is branch-context aware. It expands to `https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/<branch>`. In this case, the `<branch>` would be `master`.

```suggestion
    See {jsclient}/client-helpers.html[client.helpers.*]
```